source/read/retrieve.txt

```diff
@@ -0,0 +1,240 @@
+.. _ruby-retrieve:
+
+=============
+Retrieve Data
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: code examples, read, query, cursor
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to retrieve
+data from a MongoDB collection by using **read operations**. You can call the
+``find`` method on a collection to retrieve documents that match a set of
```

Comment:
Echoes a similar question elsewhere in the review – should I not refer to methods by name using parentheses (i.e. `find()`)?

Reply:
asked in the group slack channel!

Reply:
Parentheses are optional in Ruby as long as it isn't ambiguous. By convention, we typically omit parens for zero-argument calls (e.g. `client.close`), single-argument calls (often when the argument is a literal value, e.g. `puts "hello"`), and when the only argument is a `Hash` instance (in which case the curly braces are also optional, e.g. `Foo.new a: 1, b: 'hi', c: var`).

Reply:
Makes sense, thank you!